torchinductor/debug.py

```diff
@@ -20,6 +20,7 @@
 from torch.fx.passes.tools_common import legalize_graph
 
 import torchinductor
+from torchdynamo.config import log_level
```

Comment:
Doing the import this way means if I change the value after import you won't pick up the change.